workers/analyzer/src/main/kotlin/analyzer/AnalyzerDownloader.kt

```diff
@@ -49,10 +49,12 @@ class AnalyzerDownloader {
         val vcs = VersionControlSystem.forUrl(repositoryUrl, config)
         requireNotNull(vcs) { "Could not determine the VCS for URL '$repositoryUrl'." }
 
+        val initRevision = revision.takeUnless { it.isEmpty() } ?: vcs.getDefaultBranchName(repositoryUrl)
```

Comment:
I'm unsure about the "init" here and in `DownloadResult`, as to me it sounds more like the "definite" / resolved revision used to initialize the repository on a low level. Maybe "originalRevision" instead?

Reply:
I called it "init" because it's used to initialize the repository. "originalRevision" sound like the raw input, not like a value that might come from `getDefaultBranchName`.